tgui/packages/tgui/interfaces/SmartVend.tsx

```diff
@@ -169,6 +177,7 @@ const ItemList = ({ item }) => {
   const fallback = (
     <Icon name="spinner" lineHeight="32px" size={3} spin color="gray" />
   );
+  const [itemCount, setItemCount] = useState<number>(1);
```

Comment:
```suggestion
  const [itemCount, setItemCount] = useState(1);
```
this is already typed as a number when using this initial value

typing usestate is only valuable in specific situations, ie: selections where you can deselect something ie `<thing | undefined>` `<thing | null>` etc